uvicorn/supervisors/watchgodreload.py

```diff
@@ -129,8 +130,14 @@ def __init__(
         target: Callable[[Optional[List[socket]]], None],
         sockets: List[socket],
     ) -> None:
+        warnings.warn(
+            '"watchgod" is depreciated, you should switch '
+            "to watchfiles (`pip install watchfiles`)",
```

Comment:
Can we have the message below the `WatchGodReload` so the message gets printed on import, and not on instantiation?
```suggestion
            "to watchfiles (`pip install watchfiles`).",
```

Reply:
Sorry, I meant some lines above.

```python
class WatchGodReload:
    warnings.warn(...)
```
So I can get a warning with those lines already:
```python
from uvicorn.supervisors.watchgodreload import WatchGodReload
```

Reply:
pushed but tests will fail.

Make you could fix the warning handling in tests for this? I have no idea how you want to deal with this.

Honestly, I think the previous play where the warning was in init was better.

Reply:
I put it back on the `__init__`, sorry for the confusion here.